hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientAbstract.java

```diff
@@ -3444,11 +3444,8 @@ public void testNativeAclsForPrefix() throws Exception {
         .setStoreType(OzoneObj.StoreType.OZONE)
         .build();
 
-    // add acl
-    BitSet aclRights1 = new BitSet();
-    aclRights1.set(READ.ordinal());
     OzoneAcl user1Acl = new OzoneAcl(USER,
-        "user1", aclRights1, ACCESS);
+        "user1", EnumSet.of(READ), ACCESS);
```

Comment:
We can avoid wrapping in `EnumSet` if the set is not reused.

```suggestion
        "user1", ACCESS, READ);
```

(Also in a few other places, look for inline `EnumSet`.)